### ISSUE TITLE
QA fails due to network name when creating instance

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#436]
Originally reported of Google Code with ID 380.

```
Reproduced using revision 03be57a9925b2f268ab3466aafb7ccf3a5e45252:

---- 2013-02-19 14:32:19.108082 start Test adding a new instance via RAPI
GET /2/jobs/403
---- 2013-02-19 14:32:20.571887 time=0:00:01.463805 Test adding a new instance via RAPI
Traceback (most recent call last):
[…]
  File "…/ganeti/cli.py", line 1960, in GenericPollJob
    errors.MaybeRaise(msg)
  File "…/ganeti/errors.py", line 510, in MaybeRaise
    raise errcls(*args)
ganeti.errors.OpPrereqError: ("Network 'None' not found", 'unknown_entity')
```
Originally added on 2013-02-19 14:37:16 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed with:
commit 9394f4d1d52710e83508884a4193aab7b47a2fe1
Author: Dimitris Aragiorgis <dimara@grnet.gr>
Date:   Tue Feb 19 18:28:42 2013 +0200

    Minor fixes regarding nic.network change
    
    Make LookupNetwork() return None in case target is None. This fixes
    Issue 380. Rapi passes network=None and the lookup should not fail.
    
    Make network client aware of new nic.network.gnt-network info shows
    the IPs of each instance inside the network.  It parses nic.networks
    field of QueryInstances() output, which now is a list of uuids an
    not names.
    
    Signed-off-by: Dimitris Aragiorgis <dimara@grnet.gr>
    Reviewed-by: Helga Velroyen <helgav@google.com>

```
Originally added on 2013-02-19 17:34:32 +0000 UTC.

Mentioned Issue with GoogleCode ID 380 has been migrated to GitHub with ID #436.
Added Labels: Type-Defect
Changed State: Fixed
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
